### DISCUSSION TITLE
ET&IoT2021　出展とニュースレター発行のご案内

### DISCUSSION THREAD

**Discussion by N-YOKOI** [toppers/users-forum#51] [Announcements]
TOPPERSプロジェクトは、今週水曜日から開催される
ET6IoT2021　にリアル出展をいたします。
　　http://www.jasa.or.jp/expo/
TOPPERSプロジェクトのET6IoT出展ページはこちら
　　https://www.toppers.jp/et2021.html

コロナ禍で出展が危ぶまれる中、なんとかTOPPERS/SESSAMEパビリオン
として出展することとなりました。
ブースNo.はDA01 です。
共同出展社の皆様、ご協力くださった皆様、ありがとうございました。
会員、ユーザの皆様も、ご来場の際はぜひパビリオンにお立ち寄りください。

また、ET&IoT出展にあわせ、ニュースレターNo.41を発行しました。
　　https://www.toppers.jp/newsletter.html
最新の活動状況及びET＆IoT出展に合わせた特集記事などが掲載されて
います。
ぜひご覧ください。

皆様のご参加をお待ちいたしております。

どうぞよろしくお願いいたします。

**Comment by ykominami** [comment]
ET&IoT2021 2日目(11/18)のTOPPERSプロジェクトブース内のプレゼンスケジュールは以下の通りです。

10:00～　TOPPERSプロジェクト各WGの活動紹介
　　　　　　箱庭WG、教育WG、TECS WG、
　　　　　　ホームネットワークWG、
　　　　　　開発者会議実行委員会　など
13:30～　ユビキタスAIコーポレーションからのご紹介(1)
　　　　　　マイコンで動くGUI開発をより簡単に、よりリッチに
　 45～　ユビキタスAIコーポレーションからのご紹介(2)
　　　　　　マルチコア／マルチOS環境でのデバイス共有／コア間・OS間通信
14:00～　NCES（名古屋大学）からのご紹介（１）
　　　　　　AUTOSAR アダプティブプラットフォームに関する
　　　　　　コンソーシアム型共同研究　　　　
14:30～　NCES（名古屋大学）からのご紹介（２）
　　　　　　ダイナミックマップ2.0 の高信頼化技術に関する
　　　　　　コンソーシアム型共同研究
15:00～　TOPPERSプロジェクトの活動紹介（高田会長）
15:30～　ユビキタスAIコーポレーションからのご紹介(1)
　　　　　　マイコンで動くGUI開発をより簡単に、よりリッチに
　 45～　ユビキタスAIコーポレーションからのご紹介(2)
　　　　　　マルチコア／マルチOS環境でのデバイス共有／コア間・OS間通信

みなさまのご来場をお待ちしております。

**Comment by ykominami** [comment]
ET＆IoT2021　も最終日となりました。
本日のブース内プレゼンスケジュールについてご案内いたします。

10:00～　TOPPERSプロジェクト各WGの活動紹介
　　　　　　箱庭WG、教育WG、TECS WG、
　　　　　　ホームネットワークWG、
　　　　　　開発者会議実行委員会　など
13:00～　TOPPERS活用アイデア・アプリケーション開発コンテスト
　　　　　　授賞式　＆　受賞者プレゼン・受賞作の紹介
14:00～　ユビキタスAIコーポレーションからのご紹介(1)
　　　　　　マイコンで動くGUI開発をより簡単に、よりリッチに
　 15～　ユビキタスAIコーポレーションからのご紹介(2)
　　　　　　マルチコア／マルチOS環境でのデバイス共有／コア間・OS間通信
14:30～　NCES（名古屋大学）からのご紹介（１）
　　　　　　AUTOSAR アダプティブプラットフォームに関する
　　　　　　コンソーシアム型共同研究　　　　
15:00～　NCES（名古屋大学）からのご紹介（２）
　　　　　　ダイナミックマップ2.0 の高信頼化技術に関する
　　　　　　コンソーシアム型共同研究
15:30～　ユビキタスAIコーポレーションからのご紹介(1)
　　　　　　マイコンで動くGUI開発をより簡単に、よりリッチに
　 45～　ユビキタスAIコーポレーションからのご紹介(2)
　　　　　　マルチコア／マルチOS環境でのデバイス共有／コア間・OS間通信
16:00～　TOPPERSプロジェクトの活動紹介（高田会長）

皆様のご来場、ご参加をお待ちしております。
